tools/SdkVersionAnalyzer/Program.cs

```diff
@@ -22,9 +22,9 @@ public static int Main(string[] args)
                     return VerifyDotnetSdkVersions(directoryRoot);
                 }
 
-            case "--modify" when args.Length == 5:
+            case "--modify" when args.Length == 4:
```

Comment:
Marking this location for myself. I'll take a look how to improve, no array size changes would not waste time to debug this extra.